Library/Homebrew/extend/os/linux/dependency_collector.rb

```diff
@@ -0,0 +1,5 @@
+class DependencyCollector
+  def ant_dep(spec, tags)
+    Dependency.new(spec.to_s, tags)
+  end
```

Comment:
Sorry to be annoying and change my mind but: rather than making this in `linux` this could probably just be in `Library/Homebrew/dependency_collector.rb` and have the `os/mac` version override it as it currently does.